### PR TITLE
菜菜子恰烂钱嗷，2700真的住不起捏...

### PR DESCRIPTION
**Pull request by tuboshufps**
呜呜Rama Hotel生意不好感觉要倒闭了，希望那些工作人员不会很快被裁吧。

### CODE REVIEW

**Comment by tuboshufps** [comment]
image

**Comment by dev-soragoto** [comment]
那么多恰烂钱的办法菜菜子非选了个最离谱的……想不通（）

File: content/zh-cn/docs/srs/thailand/chettawut.md [hunk 1]
```diff
@@ -37,7 +37,9 @@ LINE：{{< shields/line "wihhospital" >}}
 医院合作酒店：大概需要 3 ~ 4 周（疫情原因回国需要更长的转运时间，可能需要在泰国休息到状态足够好才能回国，所以时间可能需要大于4周<!-- 还是我的祖国过于强大了 -->）。
 
 - [Bangkok Rama](https://g.page/BaanSiRi)
-- [dusit-srinakarinbangkok](https://goo.gl/maps/4rzty6maGqeUFz9v7)
+
+目前医院只提供院内住宿部与 Bangkok Rama 两种选项，最低价格均为每晚,{{< currency 2700 THB >}}。需要在落地泰国前全款预付住宿费用，否则不提供接送、出院之后的探视与护理等服务。
+选择院内住宿部护士每日会探视 2 次，Bangkok Rama则为护士每日探视 1 次。
```
Comment (Harriet-Muggle) [inline content/zh-cn/docs/srs/thailand/chettawut.md:43]: 住宿需要至少在医院住满30天，或按来回机票酌情增减。
因手术含病房4天，因此可以在上述基础上少4天。
目前医院在正在进行满5晚赠1晚、满20晚赠5晚活动。（附图3为机票12/3日落地、12/29日起飞，共计26晚住宿当前费用详情） dtd 2022/12/02 
ChatWithJingLiang01
ChatWithJingLiang02
ExampleFee

**Review by Harriet-Muggle — COMMENTED** [review]
Changes suggested.
Info by Jing Liang:
- Rama's price has raised to THB2700, so it won't even be a choice anymore. (Nursing care once a day in Rama while twice a day in the hospital)
- A minimal 30-day stay at the hospital is required or depends on flight ETKT. 

Also, an example for a 26-day stay (According to the flight ETKT) in a THB2900/day deluxe room (double)  has been attached.

**Comment by Harriet-Muggle** [comment]
Merged, and I will add extra info into another pull request. 

Regards, 
Harriet Li

**Pull request merged**